libs/akita/src/lib/setValueByString.ts

```diff
@@ -6,7 +6,7 @@ import { isObject } from './isObject';
  * @example
  * setValue(state, 'todos.ui', { filter: {} })
  */
-export function setValue(obj: any, prop: string, val: any) {
+export function setValue(obj: any, prop: string, val: any, replace = false) {
```

Comment:
Same here

Reply:
this function is also available in the public api, so it's potentially a breaking change